bundles/org.openhab.binding.ecobee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -315,6 +319,26 @@
 		<label>Desired Cool Range High</label>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="actualAQAccuracy" advanced="true">
+		<item-type>Number</item-type>
+		<label>Actual Air Quality Accuracy</label>
+		<state readOnly="true" pattern="%.0f"/>
```

Comment:
You should normally rather use `%d` if the value is an integer. Same for actualAQScore.

Reply:
fixed